frontend/fingertips-frontend/playwright/page-objects/pages/chartPage.ts

```diff
@@ -345,6 +356,69 @@ export default class ChartPage extends AreaFilter {
     }
   }
 
+  private async selectBenchmarkDropdownOption(
+    component: VisibleComponent,
+    selectedAreaFilters: AreaFilters
+  ) {
+    const dropdownComponent = ChartPage.benchmarkDropDownComponent;
+    const combobox = this.page
+      .getByTestId(dropdownComponent)
+      .getByRole('combobox');
+    const options = await this.getSelectOptions(combobox);
+    const upperCaseFirstCharSelectedGroup =
+      selectedAreaFilters.group.charAt(0).toUpperCase() +
+      selectedAreaFilters.group.slice(1);
```

Comment:
I've seen this done above too. Perhaps a little helper function in a util to capitaliseFirstLetter